packages/components/src/search/style.scss

```diff
@@ -10,6 +10,13 @@
 		fill: $core-grey-light-900;
 	}
 
+	.woocommerce-tag {
+		max-width: 100%;
+	}
+	.woocommerce-tag .woocommerce-tag__text {
+		max-width: calc(100% - 24px);
+	}
+
```

Comment:
These changes are duplicated in #1597, right?

Reply:
Pre ☕️ commit.  Removed so we can address separately in #1597 - thanks!